draft-ietf-cbor-7049bis.md

```diff
@@ -1597,12 +1541,11 @@ that firewalls and other security systems that decode CBOR will only
 decode in strict mode.
 
 A decoder in strict mode will reliably reject any data that could be
-interpreted by other decoders in different ways.  It will reliably
-reject data items with syntax errors ({{syntax-errors}}).  It will
-also expend the effort to reliably detect other decoding errors
-({{semantic-errors}}). In particular, a strict decoder needs to have
-an API that reports an error (and does not return data) for a CBOR
-data item that contains any of the following:
+interpreted by other decoders in different ways.  It will expend
+the effort to reliably detect invalid data items
+({{semantic-errors}}). For example, a strict decoder needs to have an
+API that reports an error (and does not return data) for a CBOR data
+item that contains any of the following:
 
 * a map (major type 5) that has more than one entry with the same key
 
```

Comment:
Thank you!  This is significant progress.

Reply:
\o/